### DISCUSSION TITLE
No me deja enviar mi dinero a binance

### DISCUSSION THREAD

**Discussion by kamilo08** [iotaledger/firefly#8121] [Help & Support]
no puedo enviar mis iostas a binance la contraseña me sale error y esa es la que guarde tengo las 24 palabras y todo pero no me deja enviar mis iostas a binance me sale error  que puedo hacer si mi contraseña no me da? como puedo recuperar mi contraseña, espero que me puedan ayudar ya que no se que mas hacer.

**Comment by begonaalvarezd** [comment]
Hola, no hay manera de recuperar tu contraseña, puedes crear un nuevo perfil importando tu mnemonic de nuevo y asignando una nueva contraseña a ese nuevo perfil
